dependency/retrieval/.gitignore

```diff
@@ -0,0 +1 @@
+retrieve
```

Comment:
If you switch to using `go run` in your `Makefile`, then you can get rid of this.

Reply:
Done.